3rdparty/openssl/CMakeLists.txt

```diff
@@ -911,12 +911,18 @@ add_enclave_library(
 
 enclave_include_directories(
   opensslcrypto PRIVATE $<BUILD_INTERFACE:${OPENSSL_BUILD_INCLUDES}>
-  $<BUILD_INTERFACE:${OPENSSL_DIR}> $<BUILD_INTERFACE:${OPENSSL_DIR}/include>
+  $<BUILD_INTERFACE:${OPENSSL_DIR}> $<BUILD_INTERFACE:${OPENSSL_DIR}/include>)
+
+enclave_include_directories(
+  opensslcrypto PUBLIC $<BUILD_INTERFACE:${OE_INCDIR}/openenclave>
   $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}/openenclave/3rdparty>)
 
```

Comment:
Should we go one more level down mentioning openssl folder explicitly? 
$<BUILD_INTERFACE:${OE_INCDIR}/openenclave/openssl>
 $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}/openenclave/3rdparty/openssl>

Reply:
This is intentional: the OpenSSL headers are included via the pattern of `include<openssl/xxx.h`, so we add the path one level above.